mate-panel/libmate-panel-applet-private/panel-applet-container.c

```diff
@@ -629,32 +625,34 @@ mate_panel_applet_container_child_set (MatePanelAppletContainer *container,
 {
 	GDBusProxy               *proxy = container->priv->applet_proxy;
 	const AppletPropertyInfo *info;
-	GSimpleAsyncResult       *result;
-
+    GTask                    *task;
 	if (!proxy)
 		return NULL;
 
 	info = mate_panel_applet_container_child_property_get_info (property_name);
 	if (!info) {
-		g_simple_async_report_error_in_idle (G_OBJECT (container),
-						     callback, user_data,
-						     MATE_PANEL_APPLET_CONTAINER_ERROR,
-						     MATE_PANEL_APPLET_CONTAINER_INVALID_CHILD_PROPERTY,
-						     "%s: Applet has no child property named `%s'",
-						     G_STRLOC, property_name);
+        g_task_report_new_error (G_OBJECT (container),
```

Comment:
Wrong indent level. Doesn't match with the line before and the line after this code block.